src/integration/pages/frontPage/crossPlatformTests.js

```diff
@@ -8,4 +8,53 @@ export default () => {
 
     expect(sect).toBeInTheDocument();
   });
+
+  it('I can see Index Alsos', () => {
+    const topStories = document.querySelector(
+      '[aria-labelledby="Top-stories"]',
+    );
+
+    if (topStories) {
+      const indexAlsos = topStories.querySelector('[data-e2e=index-alsos]');
+
+      if (indexAlsos) {
+        const h4 = indexAlsos.querySelector('h4');
+        expect(h4.textContent).toMatchSnapshot();
+
+        const data = window.SIMORGH_DATA;
+
+        if (data) {
```

Comment:
I'm only being nitpicky but could this just be `if (window.SIMORGH_DATA)` I like the context that it's the hydration data rather than a generic `data` that I have to find out what it actually is